gnomad_qc/v4/sample_qc/create_sample_qc_metadata_ht_genomes.py

```diff
@@ -0,0 +1,129 @@
+"""
+Script to create sample QC metadata HT for genomes.
+
+This script is used to create the sample QC metadata HT for genomes. The main updates
+from v3.1 to v4.0 are:
+Sample QC annotations for HGDP + TGP subset, this script directly merges the updated
+annotations from the subset meta HT to the full meta HT.
```

Comment:
```suggestion
The main updates from v3.1 to v4.0 are are due to the following updates in the HGDP + TGP subset meta:
- 169 samples added
- 110 samples removed

The 169 samples added were diverse HGDP/TGP samples in v3.1 that were unintentionally removed during sample outlier detection.

The 110 samples removed were removed due to the following reasons:
- Filtered due to contamination flagged by CHARR
- Removed due to updated relatedness results

All of the samples impacted (169 added, 110 released) are releasable; the updates we made here were to their sample QC status (high quality vs filtered).
```

it might also be helpful to add this code snippet you sent me:
```
import hail as hl

from gnomad_qc.v4.resources.sample_qc import hgdp_tgp_meta_updated

from gnomad_qc.v4.annotations.generate_freq_genomes import get_updated_release_samples

pop_diff, added, subtracted = get_updated_release_samples(hgdp_tgp_meta_updated.ht())
```

also, please update the text if I have gotten any of the details wrong